src/main/fulcro/websockets.cljs

```diff
@@ -97,4 +104,5 @@
                      :transit-handlers      transit-handlers
                      :app                   (atom nil)
                      :stop                  (atom nil)
-                     :req-params            req-params})))
+                     :req-params            req-params
+                     :sente-options         sente-options})))
```

Comment:
What happens if you don't pass options?  You'd be passing nil to sente...is that ok?

Reply:
Also you said something about "auto", but I don't see it.